src/librustc/mir/visit.rs

```diff
@@ -874,6 +879,22 @@ macro_rules! make_mir_visitor {
                 self.visit_source_scope(scope);
             }
 
+            fn super_user_type_ascription_data(
+                &mut self,
+                _ascription: UserTypeAscription,
+                data: & $($mutability)* UserTypeAscriptionData<'tcx>,
+            ) {
+                let UserTypeAscriptionData {
+                    ref $($mutability)* user_type_projection,
+                    ref $($mutability)* span,
+                    place: _, // Don't visit place..
```

Comment:
I'm not sure that it matters -- it might -- but this alters the behavior of the visitor. We used to (in `super_ascribe_user_ty`) visit the place.

Reply:
Resolving this as all the code has changed.